app/com/lynxanalytics/biggraph/controllers/FEAggregations.scala

```diff
@@ -46,26 +46,39 @@ object AttributeWithAggregator {
     attr: Attribute[T], choice: String)(
     implicit
     manager: MetaGraphManager): AttributeWithAggregator[_, _, _] = {
-
+    def d = attr.runtimeSafeCast[Double]
+    def v = attr.runtimeSafeCast[Vector[Double]]
     choice match {
       case "sum" =>
-        AttributeWithAggregator(attr.runtimeSafeCast[Double], graph_operations.Aggregator.Sum())
+        AttributeWithAggregator(d, graph_operations.Aggregator.Sum())
       case "count" => AttributeWithAggregator(attr, graph_operations.Aggregator.Count[T]())
       case "min" =>
-        AttributeWithAggregator(attr.runtimeSafeCast[Double], graph_operations.Aggregator.Min())
+        AttributeWithAggregator(d, graph_operations.Aggregator.Min())
       case "max" =>
-        AttributeWithAggregator(attr.runtimeSafeCast[Double], graph_operations.Aggregator.Max())
+        AttributeWithAggregator(d, graph_operations.Aggregator.Max())
       case "average" => AttributeWithAggregator(
-        attr.runtimeSafeCast[Double], graph_operations.Aggregator.Average())
+        d, graph_operations.Aggregator.Average())
       case "first" => AttributeWithAggregator(attr, graph_operations.Aggregator.First[T]())
       case "std_deviation" => AttributeWithAggregator(
-        attr.runtimeSafeCast[Double], graph_operations.Aggregator.StdDev())
+        d, graph_operations.Aggregator.StdDev())
       case "most_common" =>
         AttributeWithAggregator(attr, graph_operations.Aggregator.MostCommon[T]())
       case "count_most_common" =>
         AttributeWithAggregator(attr, graph_operations.Aggregator.CountMostCommon[T]())
       case "count_distinct" =>
         AttributeWithAggregator(attr, graph_operations.Aggregator.CountDistinct[T]())
+      case "elementwise_sum" =>
+        AttributeWithAggregator(v, graph_operations.Aggregator.ElementwiseSum())
+      case "elementwise_min" =>
+        AttributeWithAggregator(v, graph_operations.Aggregator.ElementwiseMin())
+      case "elementwise_max" =>
+        AttributeWithAggregator(v, graph_operations.Aggregator.ElementwiseMax())
+      case "elementwise_average" =>
+        AttributeWithAggregator(v, graph_operations.Aggregator.ElementwiseAverage())
+      case "elementwise_std_deviation" =>
+        AttributeWithAggregator(v, graph_operations.Aggregator.ElementwiseStdDev())
+      case "concatenate" =>
+        AttributeWithAggregator(v, graph_operations.Aggregator.Concatenate[Double]())
```

Comment:
Would it be hard to also support string vectors for this one? Or we don't support those anyways in other places?

Reply:
> Would it be hard to also support string vectors for this one? Or we don't support those anyways in other places?

I tried to make it work for any Vector, but couldn't figure out the type stuff. And by that point I was also of the opinion that this was a dumb idea to begin with. Why would you want to concatenate vectors in random order?

But if you see a good use case, I can have another go at it.

Reply:
I guess it's a poor man's union. As long as Vector is a poor man's set with multiplicities. :) But yeah, I wouldn't get into a jungle fight with types for this. But maybe if easy just add it for strings?

But all this reminds me: we don't even have union and intersection aggregations for sets. Maybe we should?

Alright, based on how hard you see this, either add for strings vector concat and set union and set intersection, or do nothing for now and create an issue?

Reply:
> Alright, based on how hard you see this, either add for strings vector concat and set union and set intersection, or do nothing for now and create an issue?

Created #81. Sorry.